tasks/main.yml

```diff
@@ -102,7 +102,7 @@
 
 - name: Clone OpenDroneMap repository {{ opendronemap_work_dir }}
   git:
-    repo=https://github.com/OpenDroneMap/OpenDroneMap
+    repo=https://github.com/qedsoftware/OpenDroneMap
     dest="{{ opendronemap_work_dir }}"
     version=v0.1
```

Comment:
Not necessary, module [git](http://docs.ansible.com/ansible/git_module.html) allow any kind of refs, it can be simply SHA1 for example.